.github/workflows/nightly.yml

```diff
@@ -7,26 +7,68 @@ on:
     inputs:
       ignore_commits_force_release:
         type: boolean
-        description: 'Ignore (new) commits and force a release'
+        description: 'Do not check for new commits / Force new release'
         default: false
 
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
+
+permissions:
+  contents: write
+
 jobs:
   release:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
         with:
           fetch-depth: '0'
-          ref: 'dev'
-          token: ${{secrets.ACCESS_TOKEN}}
+      
+      - uses: actions/checkout@v4
+        with:
+          repository: 'TeamNewPipe/NewPipe'
+          path: workdir
```

Comment:
I would specify `dev` as branch explicitely here, just so it's clear

Reply:
Fixed